Command/IndexCreateCommand.php

```diff
@@ -38,7 +38,13 @@ protected function configure()
                 InputOption::VALUE_NONE,
                 'If the time suffix is used, its nice to create an alias to the configured index name.'
             )
-            ->addOption('no-mapping', 'nm', InputOption::VALUE_NONE, 'Do not include mapping');
+            ->addOption('no-mapping', 'nm', InputOption::VALUE_NONE, 'Do not include mapping')
+            ->addOption(
+                'if-not-exists',
+                'ine',
```

Comment:
http://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html
With this standard, it could understand wrong the options.